frontend/src/employee-frontend/components/person-profile/PersonVoucherValueDecisions.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import { orderBy } from 'lodash'
+import { faChild } from 'lib-icons'
+import { Loading, Result } from 'lib-common/api'
+import { formatDate } from 'lib-common/date'
+import Loader from 'lib-components/atoms/Loader'
+import CollapsibleSection from 'lib-components/molecules/CollapsibleSection'
+import { Table, Tbody, Td, Th, Thead, Tr } from 'lib-components/layout/Table'
+import { useTranslation } from 'employee-frontend/state/i18n'
+import { VoucherValueDecisionSummary } from 'employee-frontend/types/invoicing'
+import { getPersonVoucherValueDecisions } from 'employee-frontend/api/invoicing'
+import { formatCents } from 'employee-frontend/utils/money'
+
+interface Props {
+  id: string
+  open: boolean
+}
+
+export default React.memo(function PersonVoucherValueDecisions({
+  id,
+  open
+}: Props) {
+  const { i18n } = useTranslation()
+
+  const [voucherValueDecisions, setVoucherValueDecisions] = useState<
+    Result<VoucherValueDecisionSummary[]>
+  >(Loading.of())
+
+  const loadDecisions = useCallback(() => {
+    setVoucherValueDecisions(Loading.of())
+    void getPersonVoucherValueDecisions(id).then((response) => {
+      setVoucherValueDecisions(response)
+    })
+  }, [id, setVoucherValueDecisions])
+
+  useEffect(loadDecisions, [loadDecisions])
+
+  return (
+    <CollapsibleSection
+      icon={faChild}
+      title={i18n.personProfile.voucherValueDecisions}
+      data-qa="person-voucher-value-decisions-collapsible"
+      startCollapsed={!open}
+    >
+      <Table data-qa="table-of-voucher-value-decisions">
+        <Thead>
+          <Tr>
+            <Th>{i18n.valueDecisions.table.validity}</Th>
+            <Th>{i18n.valueDecisions.table.number}</Th>
+            <Th>{i18n.valueDecisions.table.totalValue}</Th>
+            <Th>{i18n.valueDecisions.table.totalCoPayment}</Th>
+            <Th>{i18n.valueDecisions.table.createdAt}</Th>
+            <Th>{i18n.valueDecisions.table.sentAt}</Th>
+            <Th>{i18n.valueDecisions.table.status}</Th>
+          </Tr>
+        </Thead>
+        {voucherValueDecisions
+          .map((data) => (
+            <Tbody key="table-body">
```

Comment:
this seems a bit confusing. is the key unnecessary since it is a static value?

Reply:
Eslint incorrectly interprets that this returns an array of react elements and complains about the lack of a key. I don't know if an `// eslint-disable` would be better.